recipes/orange3/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=/dev/null
```

Comment:
I have tried this, but the build failed:
https://travis-ci.org/conda-forge/staged-recipes/jobs/144123354#L3267

It complains that xlrd is not listed in the recipe requirements (but it is). Any ideas how to make it work?